storagenode/contact/chore.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package contact
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/sync2"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/transport"
+	"storj.io/storj/satellite/overlay"
+	"storj.io/storj/storagenode/trust"
+)
+
+var mon = monkit.Package()
+
+// Config contains configurable parameters for contact chore
+// TODO: Hide MaxSleep from CLI
+type Config struct {
+	Interval time.Duration `help:"how frequently the node contact chore should run" releaseDefault:"1h" devDefault:"30s"`
+	MaxSleep time.Duration `help:"maximum duration to wait before pinging satellites" releaseDefault:"45m" devDefault:"0s"`
```

Comment:
Maybe along with hiding MaxSleep from the CLI, we can write a note in the code that encourages the SNO to keep the default to increase the likelihood of a successful connection to the satellite. To prevent people from messing around in the code.